editions/tw5.com/tiddlers/system/tw5.com-styles.tid

```diff
@@ -387,8 +387,8 @@ type: text/vnd.tiddlywiki
 
 /* WikiText rules */
 
-.multi-columns,
-.fourcolumns {
+.tc-multi-columns,
+.tc-four-columns {
```

Comment:
I meant, that `fourcolumns` should stay as it is, because it is used in the user facing text / example -- So users should **not** use the `tc-` prefix. It is reserved for TW internally.